src/machines/modelingMachine.ts

```diff
@@ -531,16 +532,16 @@ export const modelingMachine = setup({
         }
       }
     ),
-    // eslint-disable-next-line @typescript-eslint/no-misused-promises
-    'hide default planes': () => kclManager.hidePlanes(),
+    'hide default planes': () => {
+      void kclManager.hidePlanes()
+    },
```

Comment:
```suggestion
    // eslint-disable-next-line @typescript-eslint/no-misused-promises
    'hide default planes': () => kclManager.hidePlanes(),
```

I rather we leave this one until we actually move it into an actor, the `void` is equivalent to the the ignore imho